src/intersection-observer-polyfill.js

```diff
@@ -440,12 +435,11 @@ export class IntersectionObserverPolyfill {
    * @private
    */
   handleMutationObserverNotification_() {
-    if (this.mutationPass_.isPending()) {
-      return;
+    if (this.viewport_ && this.resources_) {
+      this.resources_.onNextPass(() => {
+        this.tick(this.viewport_.getRect(), undefined, true);
```

Comment:
This can cause multiple ticks to happen back to back. If `tick` is not already throttled, it should be.